rules/windows/privilege_escalation_printspooler_suspicious_file_deletion.toml

```diff
@@ -0,0 +1,53 @@
+[metadata]
+creation_date = "2021/07/06"
+maturity = "production"
+updated_date = "2021/07/06"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects deletion of print driver files by unusual process. This may indicate a clean up attempt post successful
+privilege escalation via Print Spooler service related vulnerabilities.
```

Comment:
```suggestion
Detects deletion of print driver files by an unusual process. This may indicate a clean up attempt post successful
privilege escalation via Print Spooler service related vulnerabilities.
```